lib/feedback.ts

```diff
@@ -245,7 +249,8 @@ export class Feedback {
     const data = {
       description: this._form[0].value,
       additionalinfo: this._options.additionalInfo,
-      screenshot: this._screenshotCanvas.toDataURL()
+      screenshot: this._screenshotCanvas.toDataURL(),
+      ..._.omit(this._options, this._optionsKeys) // include user custom fields
```

Comment:
Why don't we just do 

```
...this._options.additional
```

Then change the data we passed in as
```
  this._options.additional = {
    additionalInfo: xxx,
    workerGuid: xxxx,
  }
```

This way we can pass any additional data we want

Reply:
I think add custom field same level as description, screenshot is better than add it inside an object.
The code of this MR also help us pass any data we want

Reply:
I'm not quite sure I follow since we don't know what's included in the _optionsKeys, we would just remove additionalInfo (as that's the one we added previously, and replace with the additional data that we would pass in.)?

Reply:
In my logic
- _optionsKeys are contain keys of _options object, It's default options of feedback
- When user using feedback, They will add more data to options
- I using _.omit to get data that user passed, pass _optionsKeys as the keys from default options
- merge it with the data feedback build for request (screenshot, description)

Reply:
Yes, I understand, but I don't think we want to add all the keys under _options

Reply:
The value of _.omit(this._options, this._optionsKeys) is an object that contain custom fields of user.

But if you want to use just 1 field additional to contain user data, I will update it. We may choose another name because feedback has a key is additionalinfo :D

Reply:
additionalinfo is added by us previously, so we can change it to anything we want really.

Or we can just pass this additionalinfo as an object instead of a string as well?

Reply:
Change type if additionalinfo may cause the issue in our API while old client passed it as string but new client pass it as object.